client/packages/system/src/Stock/Components/NewStockLineModal.tsx

```diff
@@ -33,11 +33,12 @@ export const NewStockLineModal: FC<NewStockLineModalProps> = ({
 
   const { Modal } = useDialog({ isOpen, onClose });
 
-  const { draft, setDraft, create } = useStockLine();
-
-  const onUpdate = (patch: Partial<DraftStockLine>) => {
-    setDraft({ ...draft, ...patch });
-  };
+  const {
+    query: { isLoading },
+    draft,
+    updatePatch,
+    create: { create },
```

Comment:
Sorry can't put comment on the `save` method below, but wondering if a redirect to the detail page would be nicer than just the toast popup on save?

Maybe worth asking for UX input, but given we don't want `New Stock` to be used often/to add lots of stock lines in one go, I think it's nice to be redirected so you can see your saved changes?

Found myself searching through the stock list trying to find the stock line I just created, hoping it worked 😅

Reply:
> Found myself searching through the stock list trying to find the stock line I just created, hoping it worked 😅

Me too!

Good idea: d64e534d4